README.md

```diff
@@ -145,7 +151,17 @@ curl -X POST http://localhost:11434/api/generate -d '{
 }'
 ```
 
-## Tools using Ollama
+<hr/>
+
+> The Rest API is currently designed to work on localhost (since CORS is implemented for localhost only).
```

Comment:
I would add a small subsection here – and note that you don't need `OLLAMA_PORT` anymore, you can just add the port to `OLLAMA_HOST` 😊 

Something like:

### Remote

To connect to Ollama on another host, use `OLLAMA_HOST`:

```
OLLAMA_HOST=192.168.1.24:11434 ollama run llama2
```